public/locales/en-US.json

```diff
@@ -281,35 +314,15 @@
     "placeholder_uom": "Select unit",
     "placeholder_usage": "Select usage",
     "placeholder_filter": "Filter by",
+    "placeholder_filter_architecture": "Filter by architecture",
     "placeholder_filter_billing_provider": "Filter by purchased through",
     "placeholder_filter_displayName": "Filter by name",
     "placeholder_filter_granularity": "Filter by date",
     "placeholder_filter_rangedMonthly": "Filter by month",
     "placeholder_filter_sla": "Filter by SLA",
     "placeholder_filter_uom": "Filter by unit",
     "placeholder_filter_usage": "Filter by usage",
-    "billing_provider": "None",
-    "billing_provider_aws": "Amazon Web Services",
-    "billing_provider_azure": "Microsoft Azure",
-    "billing_provider_gcp": "Google Cloud Platform",
-    "billing_provider_oracle": "Oracle Cloud Infrastructure",
-    "billing_provider_red hat": "Red Hat Marketplace",
-    "granularity_Daily": "Daily",
-    "granularity_Weekly": "Weekly",
-    "granularity_Monthly": "Monthly",
-    "granularity_Quarterly": "Quarterly",
-    "granularityRangedMonthly": "{{context}}",
-    "granularityRangedMonthly_current": "This month",
```

Comment:
required:
looks like we missed these two conversions in `dateHelpers.js`

line 100
```
dateTime.title = translate('curiosity-toolbar.label', { context: ['granularityRangedMonthly', dateTime.title] });
```

and line 111
```
title: translate('curiosity-toolbar.label', { context: ['granularityRangedMonthly', 'current'] })
```